integration-tests/gradle-plugin-test/multi-platform/src/nativeTest/kotlin/io/realm/test/multiplatform/util/platform/PlatformUtils.kt

```diff
@@ -15,8 +15,11 @@
  * limitations under the License.
  */
 
+@file:OptIn(ExperimentalForeignApi::class)
```

Comment:
Do we need this if you added it to the freeCompilerArgs?

Reply:
But it is not in the `freeCompilerArgs` ... for this project ;) 

I just added it like this for this `integration-test` project as it was immediately supported through the IDE compare to the `freeCompilerArgs` approach. 

The reason why I used the `freeCompilerArgs` for our library and test modules are that I considered it tedious to have to scatter them across a lot of files. I also think that our project uses these APIs so heavily that we don't want to consider it on a as-needed basis.

Reply:
Ah, yeah, missed the full path 🙈